minerva-core/src/main/java/org/geneontology/minerva/BlazegraphMolecularModelManager.java

```diff
@@ -598,7 +603,8 @@ protected OWLOntology loadModelABox(IRI modelId) throws OWLOntologyCreationExcep
 				graphs.close();
 				RepositoryResult<Statement> statements =
 						connection.getStatements(null, null, null, false, new URIImpl(modelId.toString()));
-				OWLOntology abox = loadOntologyDocumentSource(new RioMemoryTripleSource(statements), true);
+				boolean minimal = false;
```

Comment:
`minimal` is supposed to be `true` here I think. This method loads only the abox without needing the tbox, but needs to be used only for RDF use cases, like for pulling out model metadata or loading into Arachne.